.github/workflows/test.yaml

```diff
@@ -11,3 +11,42 @@ jobs:
       self-hosted-runner: false
       self-hosted-runner-label: "edge"
       working-directory: ./bind-operator
+
+  django-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.2.2
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          cd dns-policy-app/app
+          python3 -m pip install --upgrade pip
+          pip install -r requirements.txt
+          docker run --rm --name db -d -e POSTGRES_PASSWORD=testpassword postgres:17.2
```

Comment:
Would be possible to use services?
https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services

Reply:
I tried and unit tests don't run when I use a service. I think those are limited to tests running in containers

Reply:
Actually, it was the `working-directory` directive which was faulty (it's a operator-workflows thing and not native to github). So this is working now